source/gitlab/gitlab.go

```diff
@@ -153,6 +173,13 @@ func (g *Gitlab) readDirectory() error {
 	return nil
 }
 
+func (g *Gitlab) ensureFields() {
```

Comment:
Likewise, this should be called in `Open()` or `WithInstance()` instead of `Gitlab` methods that act on a `Gitlab` instance